features/zoek-met-burgerservicenummer/gba/actuele-reisdocumenten-gba.feature

```diff
@@ -0,0 +1,72 @@
+#language: nl
+
+@gba
+Functionaliteit: zoeken van de actuele reisdocumenten van een persoon
+
```

Comment:
Ik zou hier een korte beschrijving opnemen wat er met de functionaliteit kan worden bereikt, bijv.

Als afnemer
Wil ik de reisdocumenten kunnen bevragen van een persoon
Zodat ik kan zien welke reisdocumenten hij volgens de registratie in zijn bezit heeft

Reply:
overgenomen